gradle.properties

```diff
@@ -19,3 +19,5 @@ android.useAndroidX=true
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
You can remove this line: its usage it deprecated and you don't use build config fields if I'm correct

Reply:
Done: https://github.com/didomi/sample-app-android-java/pull/3/commits/08332823e7109d915e0637650af408327cc307f2